CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
    * REMOVED: needs_ci_run script [#4423](https://github.com/valhalla/valhalla/pull/4423)
    * REMOVED: unused vehicle types in AutoCost and segway; renamed kTruck to "truck" instead of "tractor_trailer" [#4430](https://github.com/valhalla/valhalla/pull/4430)
 * **Bug Fix**
+   * FIXED: Config singleton multiple instantiation issue [#4521](https://github.com/valhalla/valhalla/pull/4521)
```

Comment:
i know its slightly confusing but please move this to the bottom of the bug fix list. we order ours from oldest at the top to newest at the bottom of each list. sorry for the annoyance

Reply:
done